pkg/invoke/exec.go

```diff
@@ -52,6 +53,7 @@ func ExecPluginWithResult(pluginPath string, netconf []byte, args CNIArgs) (*typ
 	return res, err
 }
 
+// ExecPluginWithoutResult invokes network plugin but ignore the results.
```

Comment:
but ignores the results. It returns any error encountered while executing the plugin